### PR TITLE
First pass at WKO migration doc

### PR DESCRIPTION
**Pull request by rosemarymarano**
This is a first pass at the Cert-Manager Migration Guide.

### CODE REVIEW

File: content/en/docs/guides/migrate/wko.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+
+```
+$ vz export oam --name <app-name> --namespace <app-namespace> > myapp.yaml
+```
+</div>
+{{< /clipboard >}}
+
+This generates a YAML file for the OAM application in `myapp.yaml`. Make any local customizations to the generated YAML file, and then apply the YAML file to the OCNE 2.0 environment by continuing to follow the documentation.
+
+Note that, for each OAM application, the WebLogic Kubernetes Operator requires an Istio Authorization Policy to communicate with the WebLogic servers. The required Authorization Policies are created for you by the `vz export oam`  command, as documented here: https://verrazzano.io/devel/docs/guides/migrate/oam-to-kubernetes/ingress-trait/.
```
Comment (bobdonat) [inline content/en/docs/guides/migrate/wko.md:219]: It is not clear to me what an ingress trait has to do with the WebLogic Kubernetes Operator and it's required Istio authorization policy.

Comment (mgianatagh) [reply content/en/docs/guides/migrate/wko.md:219]: I would be ok with dropping this paragraph.

Comment (marcogelle) [reply content/en/docs/guides/migrate/wko.md]: I removed the paragraph.

File: content/en/docs/guides/migrate/wko.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    matchLabels:
+       app: weblogic-operator
+  policyTypes:
+  - Ingress
+EOF
+```
+</div>
+{{< /clipboard >}}
+
+## Migrate WebLogic Kubernetes Operator with OAM applications to OCNE 2.0
```
Comment (mgianatagh) [inline content/en/docs/guides/migrate/wko.md:203]: A better title may be "Migrate OAM WebLogic applications to OCNE 2.0"

**Review by mgianatagh — APPROVED** [review]

**Review by usabada — APPROVED** [review]

**Pull request merged**